ui/admin/app/templates/scopes/scope/targets/target/add-egress-worker-filter.hbs

```diff
@@ -0,0 +1,35 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.target.workers.actions.add-egress-worker-filter')}}
+<Breadcrumbs::Item
+  @text={{t 'resources.target.workers.actions.add-egress-worker-filter'}}
+  @route='scopes.scope.targets.target.add-egress-worker-filter'
+/>
+
+<Rose::Layout::Page as |page|>
+
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+  <page.header>
+    <h2>
+      {{t 'resources.target.workers.actions.add-egress-worker-filter'}}
+      <DocLink @doc='target.worker-filters' @iconSize='large' />
+    </h2>
+    <p>{{t 'resources.target.workers.ingress-workers.description'}}</p>
+  </page.header>
+
+  <page.body class='target-workers'>
+    <Form::Target::WorkerFilter
+      @name='egress_worker_filter'
+      @model={{@model}}
+      @onInput={{fn (mut @model.egress_worker_filter)}}
```

Comment:
I don't think we should be using mut anymore, it's been discouraged [now](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-mut-helper.md)

Reply:
Ah totally missed that, thank you for the link